docs/how_to/install_puppet_enterprise.md

```diff
@@ -0,0 +1,107 @@
+# How-To Install Puppet Enterprise
```

Comment:
@tvpartytonight this isn't a reference, because it's in no way complete. 'How To' guides are a major section of our docs structure, and I've been trying to create a pattern of the path & title matching up in new docs, like in this one. Would removing the hyphen make it better in the title?